### PR TITLE
Translate the topic list

### PR DESCRIPTION
**Pull request by GeBo1**
Did the best I could on this.

- [Bingata](https://en.wikipedia.org/wiki/Bingata)
- [Shisha](https://en.wikipedia.org/wiki/Shisa)

Maybe `Treeing` should be `Tree Climbing`?

And I'm not at all certain on this one:
```
ウムイの鐘=Umui Bells 
```

### CODE REVIEW

**Comment by ManlyMarco** [comment]
> Maybe `Treeing` should be `Tree Climbing`?

Yes.

> ウムイの鐘=Umui Bells 

It's the bell in a circle thing near the lighthouse, like [this one](http://amamishimbun.co.jp/wp-content/uploads/5957f73d8bacf2b75fb3aa83ef987389.jpg). Not sure how to call it in english.

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
+初経験=First Experience
+ぬくもり=Warmth
+体の相性=Body Chemistry
+甘い夜=Sweet Nights
+胸の話=Breast Stuff
+お尻の話=Butt Stuff
+性器=Genitals
+フェチ=Fetishes
+好きな体位=Favorite Position
+欲求不満=Frustration
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:51]: Sexual frustration (desire)

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
+マンタ=Manta Ray
+ウミガメ=Sea Turtles
+海ほたる=Sea Fireflies
+星の砂=Star Sand
+お土産=Souvenirs
+名物料理=Specialty Dishes
+シークヮーサージュース=Shikuwasa juice
+ぬいぐるみ=Stuffed Animals
+ウムイの鐘=Umui Bells
+紅型=Bingata
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:38]: https://ja.wikipedia.org/wiki/紅型 Traditional dyed cloth from Okinawa

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
+三線=Shamisen
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:39]: https://ja.wikipedia.org/wiki/三線 Okinawan traditional three-stringed instrument

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt [hunk 4]
```diff
@@ -1,51 +1,51 @@
...
+シーサー=Shisa
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:40]: Legendary statue of a beast

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt [hunk 5]
```diff
@@ -1,51 +1,51 @@
...
+キスの味=Taste of Kisses
+初経験=First Experience
+ぬくもり=Warmth
+体の相性=Body Chemistry
+甘い夜=Sweet Nights
+胸の話=Breast Stuff
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:46]: Breast talk

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt [hunk 6]
```diff
@@ -1,51 +1,51 @@
...
+お尻の話=Butt Stuff
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:47]: Ass talk

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt [hunk 7]
```diff
@@ -1,51 +1,51 @@
...
+サンゴ=Coral
+クマノミ=Clownfish
+チンアナゴ=Spotted Garden Eel
+マンタ=Manta Ray
+ウミガメ=Sea Turtles
+海ほたる=Sea Fireflies
+星の砂=Star Sand
+お土産=Souvenirs
+名物料理=Specialty Dishes
+シークヮーサージュース=Shikuwasa juice
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:35]: https://en.wikipedia.org/wiki/Citrus_depressa
Citrus depressa (Citrus × depressa, formerly C. pectinifera, Okinawan: シークヮーサー/シークァーサー shiikwaasa, Japanese: ヒラミレモン hirami remon or シークワーサー shīkuwāsā), in English sometimes called shiikuwasha, shequasar, Taiwan tangerine, Okinawa lime,[1] flat lemon, hirami lemon, or thin-skinned flat lemon

**Review by ManlyMarco — COMMENTED** [review]
Have fun with figuring how to name these

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt (hunk 2 above)
Comment (GeBo1) [reply Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:38]: Oh, yeah, I knew that, but forgot the link

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt (hunk 1 above)
Comment (GeBo1) [reply Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:51]: They're color coded as well in game, so it's obvious what it is.  In context it seems much more frustration than desire.

File: Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt (hunk 2 above)
Comment (GeBo1) [reply Translation/en/RedirectedResources/assets/abdata/action/list/topic/00/00/translation.txt:38]: Changed to be specific and singular until we have something better

**Pull request merged**